### ISSUE TITLE
Add ability to purge remote content and search index

### ISSUE THREAD

**Issue by pullopen** [Plume-org/Plume#992]
**Is your feature request related to a problem? Please describe.**
I notice that after a while, the search_index and database file grows bigger and bigger, filled with remote article that we usually do not need or understand.

**Describe the solution you'd like**
I would like to have a `plm` order to purge remote content as well as the related search index.

**Describe alternatives you've considered**
Or would you please give me some instruction?

**Comment by trinity-1686a** [comment]
for the search_index, you can (or a future sub-command can) simply delete the search index, and recreate it with `plm`. However it won't have much of an effect if remote content still exist in the database as it will just get reindexed

**Comment by pullopen** [comment]
> for the search_index, you can (or a future sub-command can) simply delete the search index, and recreate it with `plm`._ However _it won't have much of an effect if remote content still exist in the database as it will just get reindexed

Yes, so I'm asking if there's a way to safely delete remote content in database without affecting the local one.
